scripts/rev_billing.php

```diff
@@ -0,0 +1,63 @@
+<?php
+if (file_exists(dirname(__FILE__).'/../config.inc.php')) {
+    require_once dirname(__FILE__).'/../config.inc.php';
+} else {
+    require_once dirname(__FILE__).'/../config.sample.php';
+}
+
+//Establish mediahub connection
+$media_hub = new UNL_MediaHub();
+
+$db = Doctrine_Manager::getInstance()->getCurrentConnection();
+
+//Get all orders that have not been completed.
+$orders = new UNL_MediaHub_RevOrderList(
+    array(
+        'for_billing'=>true,
+        'after_date'=> date('Y-m-d H:i:s', strtotime('-1 month'))
```

Comment:
Is this intended to be run _extactly_ at the same time every month? Is there a concern that orders with be reconciled/billed multiple times, or worse, be missed.